tests/e2e/utils/flows.js

```diff
@@ -198,7 +198,15 @@ export const shopperWCP = {
 			'#billing-address_1',
 			customerBillingDetails.addressfirstline
 		);
+		await clearAndFillInput(
+			'#billing-country .components-form-token-field__input',
```

Comment:
@deepakpathania we chatted about the selectors for the fields over [here](https://github.com/Automattic/woocommerce-payments/pull/3277#discussion_r758953335). Were you referring to something like this?

Reply:
Yes, having a direct identifier for input fields would have helped be more readable in scenarios like these where we want to pick them. Also avoids any second order effects of renaming any parent level identifiers in future. Might not be apt for this PR itself though but something we should try to communicate more often when introducing new fields imo.